### ISSUE TITLE
Use IP address as identifier when hostname is empty in the timeline

### ISSUE THREAD

**Issue by febrezo** [cyb3rfox/Aurora-Incident-Response#53]
Currently, the event system and the remote system column use only the hostname as the system identifier. When you only know the IP address, it has to be manually copied in both fields. If the hostname is empty, maybe using the IP address could be useful.

**Comment by cyb3rfox** [comment]
Good idea, or maybe have both in the dropdown. i'll check what works better.
